docs/sources/reference/commandline/cli.md

```diff
@@ -1681,6 +1683,13 @@ will automatically create this directory on the host for you. In the
 example above, Docker will create the `/doesnt/exist`
 folder before starting your container.
 
+    $ sudo docker run -v /icanwrite --readonly busybox touch /icanwrite here
```

Comment:
Minor nit, but `docker run --readonly -v /icanwrite ...` could be better, as "icanwrite readonly" sounds weird.